builtin/game/misc.lua

```diff
@@ -248,3 +248,36 @@ end
 function core.close_formspec(player_name, formname)
 	return minetest.show_formspec(player_name, formname, "")
 end
+
+-- Returns the exact coordinate of a pointed surface
+function minetest.get_face_pos_from_pointed_thing(placer, pointed_thing)
```

Comment:
I prefer 'pointed thing to face pos', shorter and consistent with othr functions.